bouncer/views.py

```diff
@@ -39,6 +39,9 @@ def annotation(self):
             statsd.incr("views.annotation.404.annotation_not_found")
             raise httpexceptions.HTTPNotFound(_("Annotation not found"))
 
+        if ('deleted' in document['_source'] and document['_source']['deleted']):
```

Comment:
Might be easier and more readable to do `if document.get('deleted', False) is True`.
The current implementation, for example, would raise a 404 error when the annotation includes: `{'deleted': 'nope'}`, or any other truthy value.